docs/snippets/ov_common.cpp

```diff
@@ -71,7 +71,7 @@ int main() {
     //! [ov_api_2_0:get_inputs_outputs]
 
     //! [ov_api_2_0:compile_model]
-    ov::runtime::ExecutableNetwork exec_network = core.compile_model(network, "CPU");
+    ov::runtime::CompiledModel exec_network = core.compile_model(network, "CPU");
```

Comment:
Suggest to have proper variable name as well in all places

Reply:
Fixed for samples and docs